source/install.txt

```diff
@@ -91,21 +91,22 @@ Select the appropriate tab for your operating system:
       Considerations for Ubuntu
       ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-      - ``mongosh`` is available as a
+      - :binary:`mongosh` is available as a
         :abbr:`PPA (Personal Package Archive)` for Ubuntu 20.04 (Focal)
-        and Ubuntu 18.04 (Bionic).
+        and Ubuntu 18.04 (Bionic). If you are installing on Ubuntu
+        20.04 (Focal) or Ubuntu 18.04  (Bionic), use the procedure in
+        the ``.deb`` tab.
         
-        - If you are using Ubuntu 20.04 (Focal) or Ubuntu 18.04
-          (Bionic), use the procedure in the ``.deb`` tab.
-        
-        - If you are using Ubuntu 16.04 (Xenial), you must
-          obtain the ``mongosh`` tarball using the procedure in the
-          ``.tgz`` tab.
+      - MongoDB 5.0 removes support for Ubuntu 16.04 (Xenial). If you
+        must use the unsupported Ubuntu 16.04 (Xenial), install the
+        :binary:`mongosh` tarball using the procedure in the ``.tgz``
+        tab.
```

Comment:
[question] Should we be telling users how to install mongosh on unsupported platforms? It's a little confusing to me why we aren't just telling them they can't install mongosh on Xenial.